### ISSUE TITLE
[ERROR] Entrypoint file not found at lib\main.dart

### ISSUE THREAD

**Issue by angelhdz** [flutter/flutter-intellij#4242]
Thanks for the feedback! If your issue is related to the Flutter framework itself,
please open an issue at
[github.com/flutter/flutter](https://github.com/flutter/flutter/issues/new).

## Steps to Reproduce

_Please tell us what you were doing and what went wrong_

## Version info
42.2.2

Please paste the output of running `flutter doctor -v` here (available from the command
line or from `Tools > Flutter > Flutter Doctor`). It will provide the version of the
Flutter framework as well as of the IntelliJ plugin.

![image](https://user-images.githubusercontent.com/10191085/71422569-809dee00-2659-11ea-8605-5509d538f920.png)

I can't choose the main.dart either in the lib folder. The files doesn't appear and if I type the absolute path to the main.dart, error still happens.

### More Info

My project works in VSCode, that means I have no issues with Flutter or Dartk SDKs or the JDK. ALl the environment variables/paths are ok.

**Comment by pq** [comment]
Hmmm...  I wonder if the project isn't improperly configured.  What do you see when you open Project Structure and look at the Module entries under Project Settings?

**Comment by angelhdz** [comment]
I just hit on File > New > Project and select Flutter project, then select the path to the Flutter SDK, type a name for the project and It generates all the project structure including a default debug configuration.

Flutter and Dart SDKs are properly configured in the Settings > Languages & Frameworks.

Flutter and Dart plugins are installed, latest versions. IntelliJ IDEA also latest version.

It complains that lib/main.dart isn't a Dart file... also, I can't select the main.dart file when I select it in the Edit Configuration dialog, Dart entrypoint field. I click on the folder icon to select a main.dart file, but I can only navigate to the lib folder, and no *.dart file appear to be selected.

The path to the main.dart file is correct: C:\Users\Angel\Desktop\test_flutt\lib\main.dart

test_flutt is the Flutter project I created.

I've tried so many things since I installed intelliJ to try Flutter. I even created a Dart project and added a flutter project as a module.

I've selected many versions of the Java JDK.

I'm about to get insane!

Added these libraries:

![image](https://user-images.githubusercontent.com/10191085/71543625-80e50480-294b-11ea-98a3-e8225f2a8b10.png)

No success.

This is the main project structure:

![image](https://user-images.githubusercontent.com/10191085/71543634-9c500f80-294b-11ea-8fce-b063e73748e6.png)
